Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/lucene/impl/LuceneUtility.java

```diff
@@ -596,19 +599,19 @@ private void buildQuery(BooleanClause.Occur occur, PerFieldAnalyzerWrapper perFi
             String searchString, boolean isSearchOnDescription) {
         if (luceneFieldAnnotation.isTokenized()) {
             BooleanQuery.Builder tokenizedQueryBuilder = new BooleanQuery.Builder();
-            buildTokenizedOrWildCardQuery(luceneFieldAnnotation, searchString, qBuild, tokenizedQueryBuilder);
+            buildTokenizedOrWildCardQuery(luceneFieldAnnotation, searchString, qBuild, tokenizedQueryBuilder, isSearchOnDescription);
 
             if (luceneFieldAnnotation.isShortPhraseSearchable()) {
                 BooleanQuery.Builder rootQueryBuilder = new BooleanQuery.Builder();
                 rootQueryBuilder.add(tokenizedQueryBuilder.build(), Occur.SHOULD);
                 // Short word searching
                 BooleanQuery.Builder shortWordSearchQueryBuilder = new BooleanQuery.Builder();
-                shortWordSearchQueryBuilder.add(new PhraseQuery(luceneFieldAnnotation.lucenefieldName() + LuceneFieldNames.UNTOKENIZED_SUFFIX, searchString), Occur.SHOULD);
+                shortWordSearchQueryBuilder.add(new PhraseQuery(luceneFieldAnnotation.lucenefieldName() + LuceneFieldNames.UNTOKENIZED_SUFFIX, searchString), Occur.MUST);
                 
                 /*If the searchDescription is enabled, add the query along with Word Query.
                  * Rightnow isShortPhraseSearchable is enabled only for Word field which avoids multiple entry of search params */
                 if(isSearchOnDescription) {
-                    shortWordSearchQueryBuilder.add(new PhraseQuery(SearchFieldNames.DESCRIPTION, searchString), Occur.SHOULD);
+                    shortWordSearchQueryBuilder.add(new PhraseQuery(SearchFieldNames.DESCRIPTION, searchString), Occur.MUST);
```

Comment:
why is this `MUST`? Shouldn't it be 'or', which is `SHOULD` I believe?

Reply:
I believe SHOULD or MUST will not alter the results as we another query combining the word and description query with 'SHOULD'.
``` rootQueryBuilder.add(shortWordSearchQueryBuilder.build(), Occur.SHOULD); ```

Reply:
why are you changing it then?